### PR TITLE
Add org.vertcoin.vertcoin-qt

### PR DESCRIPTION
**Pull request by vertiond**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by vertiond** [comment]
bot, build org.vertcoin.vertcoin-qt

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64", "arm", "aarch64"]
```
Comment (hfiguiere) [inline flathub.json:2]: you can remove this it's not needed.

File: org.vertcoin.vertcoin-qt.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/vertcoin-project/vertcoin-core/vertcoin-0.18.0-x86_64-linux-gnu.tar.gz",
+          "sha256": "6ded7ea883b6cf9cee95701b13eef2e601a85f91d15f255d4fc7b25db92808ec"
+        },
+        {
+          "type": "archive",
+          "only-arches": ["arm"],
+          "url": "https://github.com/vertcoin-project/vertcoin-core/vertcoin-0.18.0-arm-linux-gnueabihf.tar.gz",
+          "sha256": "cec7564b17b9e48b0f0417c929afb59f9fe1a935da64fcc3715ef33228b36a7d"
+        },
```
Comment (hfiguiere) [inline org.vertcoin.vertcoin-qt.json:43]:
```suggestion
```

this arch isn't supported anymore for at least 2 years

File: org.vertcoin.vertcoin-qt.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/vertcoind",
+        "install -m 0744 -D -t /app/bin/internal bin/vertcoin-qt",
+        "install -m 0744 -D -t /app/bin/internal bin/vertcoin-cli"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/vertcoin-project/vertcoin-core/vertcoin-0.18.0-x86_64-linux-gnu.tar.gz",
```
Comment (hfiguiere) [inline org.vertcoin.vertcoin-qt.json:35]: this URL is 404

File: org.vertcoin.vertcoin-qt.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        {
+          "type": "archive",
+          "only-arches": ["arm"],
+          "url": "https://github.com/vertcoin-project/vertcoin-core/vertcoin-0.18.0-arm-linux-gnueabihf.tar.gz",
+          "sha256": "cec7564b17b9e48b0f0417c929afb59f9fe1a935da64fcc3715ef33228b36a7d"
+        },
+        {
+          "type": "archive",
+          "only-arches": ["aarch64"],
+          "url": "https://github.com/vertcoin-project/vertcoin-core/vertcoin-0.18.0-aarch64-linux-gnu.tar.gz",
```
Comment (hfiguiere) [inline org.vertcoin.vertcoin-qt.json:47]: this URL is 404

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
also both the .desktop file and appdata should be submitted upstream to be managed there.

**Comment by vertiond** [comment]
Looks good to me.

**Comment by barthalion** [comment]
bot, build org.vertcoin.vertcoin-qt

**Pull request closed without merging**